src/conversions/either.rs

```diff
@@ -168,8 +172,10 @@ mod tests {
     use std::borrow::Cow;
 
     use crate::exceptions::PyTypeError;
-    use crate::{Python, ToPyObject};
+    use crate::Python;
 
+    use crate::prelude::IntoPyObject;
```

Comment:
I guess the IDE picked this import? Another argument for it to be exported at the top-level, perhaps.

Reply:
Jep, that was an auto-import 🙃

Reply:
While I was there I changed all the others as well